### ISSUE TITLE
mutiple proto files in docker-compose doesn't work

### ISSUE THREAD

**Issue by mtrtm** [tokopedia/gripmock#50]
I am trying to use both flag and non-flag args in docker-compose.  It works just fine until I need more than one proto file.  

Example of working docker-compose.yml file:
```
  grpc-server-example-works:
    image: tkpd/gripmock:latest
    command: -stub=/stub /proto/greetV1.proto
```  

Fails, with a log message of "File does not reside within any path specified using --proto_path (or -I).  You must specify a --proto_path which encompasses this file..."  
```
  grpc-server-example-fails:
    image: tkpd/gripmock:latest
    command: -stub=/stub /proto/greetV1.proto /proto2/server2Greeting.proto
```  
I am wondering if I am doing something wrong, or if not and it's an issue with go a simple solution would be to accept comma-separated protofiles with a named arg.

**Comment by mtrtm** [comment]
In my previous example I tried proto files in two different folders.  I decided to try two different proto files in the same folder as outlined in https://github.com/tokopedia/gripmock/blob/master/example/multi-package/entrypoint.sh

I then get 
```
Starting GripMock
2020/06/01 20:11:22 Error when reading file test.sh. invalid character 'g' looking for beginning of value. skipping...
Serving stub admin on http://:4771
2020/06/01 20:11:22 protoc-gen-go: error:inconsistent package names: greetV1, greetV2
--go_out: protoc-gen-go: Plugin failed with status code 1.
2020/06/01 20:11:22 Fail on protoc exit status 1
```

**Comment by mtrtm** [comment]
I have now built a VERY simple example with the same package name in both proto files and both proto files in the same folder just to see if I could get gripmock to serve two calls.  I am getting the following error:
```
 Failed to generate server formatting: 45:7: expected 'IDENT', found 'struct'
```
it then outputs the go file.  The last line I copied (there's more) has a struct: 
```
// Code generated by GripMock. DO NOT EDIT.
package main

import (
        "bytes"
        "encoding/json"
        "fmt"
        "io"
        "io/ioutil"
        "log"
        "net"
        "net/http"

        "github.com/mitchellh/mapstructure"
        "golang.org/x/net/context"
        "google.golang.org/grpc"
        "google.golang.org/grpc/reflection"
)

const (
        TCP_ADDRESS  = ":4770"
        HTTP_PORT = ":4771"
)



type Greeter5 struct{}





func (s *Greeter5) SayHello(ctx context.Context, in *HelloRequest5) (*HelloReply5,error){
        out := &HelloReply5{}
        err := findStub("Greeter5", "SayHello", in, out)
        return out, err
}







type  struct{}
...
```  
I would love to use this project for my teams, but until we can serve multiple proto files from different folders on one mock server, and until those files can have a different package name, we cannot.  

docker compose:
```
...
 grpc-server-example-fails:
    command: -stub=/stub /proto/greet4.proto /proto/greet5.proto
    volumes:
      - type: bind
        source: ./testprotos
        target: /proto
...
```  
greet4.proto:
```
syntax = "proto3";

package greet;

service Greeter4 {
  // Sends a greeting
  rpc SayHello4 (HelloRequest4) returns (HelloReply4);
}

// The request message containing the user's name.
message HelloRequest4 {
  string name = 1;
}

// The response message containing the greetings.
message HelloReply4 {
  string message = 1;
}
```  
greet5.proto
```
syntax = "proto3";

package greet;

service Greeter5 {
  rpc SayHello5 (HelloRequest5) returns (HelloReply5);
}

// The request message containing the user's name.
message HelloRequest5 {
  string name = 1;
}

// The response message containing the greetings.
message HelloReply5 {
  string message = 1;
}
```

**Comment by phoe3nix** [comment]
hi @mtrtm ! I took commit in this pull request: https://github.com/tokopedia/gripmock/pull/45. And this commit solve problem for me.

**Comment by mtrtm** [comment]
> hi @mtrtm ! I took commit in this pull request: #45. And this commit solve problem for me.

So you got more than one proto file to work?  Did each proto file have a different package?

I think the author is still waiting on someone to add tests around that PR

**Comment by phoe3nix** [comment]
@mtrtm I have 6-7 proto file and they in two different packages.  I have many other problems, for example cycle imports, because sed change package name of all files to main.

**Comment by mtrtm** [comment]
when is this PR going to be accepted?  I'd like to start using this

**Comment by jekiapp** [comment]
Hi @phoe3nix is it possible to add testcase in the current example test cases, so we have clear picture what is the problem and how this PR #45 solve it.

**Comment by mtrtm** [comment]
I'd love if someone could do this.  I think the only thing that would need to be added (or change the existing?) is just to make the package name be different. @jekiapp @phoe3nix

**Comment by jekiapp** [comment]
@mtrtm PR #45 merged after I made tests for it :smile: 
@phoe3nix do you have any issue with sed? feel free to open a new issue for it.

**Comment by phoe3nix** [comment]
Hi @jekiapp 
Generate server was failed, when proto files have cycled between them. It's a problem of proto files, of course. But I think, that change all packages name to `main` isn't good idea, when you start server with 5-10 different packages)

**Comment by jekiapp** [comment]
> change all packages name to `main` isn't good idea
do you have any suggestion for this?

anyway, I think it's a whole different issue, if you could open an issue and provide some example with more information, together we could find the solution :slightly_smiling_face:

**Comment by jekiapp** [comment]
I just built `tkpd/0.8.2` docker image with patch from #45 and #52 please pull and try it.
